otel/README.md

```diff
@@ -0,0 +1,53 @@
+### Steps to run this sample:
+1) Run a [Temporal service](https://github.com/temporalio/samples-go/tree/main/#how-to-use).
+
+One way could be just to use the temporal CLI.  
+
+```bash
+temporal server start-dev --ui-port 8089
+```
+
+2) Run the following command to start the worker
+```bash
+go run otel/worker/main.go
+```
+3) Run the following command to start the example
+```bash
+go run otel/starter/main.go
+```
+
+The example outputs the traces in the stdout, both the worker and the starter.  
+[Trace output](./trace.json) collects the spans generated. Please note that in order to have the valid json they have been put in a JSON array but they are returned as single JSON objects.  
```

Comment:
I don't think we need to put the pre-generated trace JSON in the repo

Reply:
I was thinking this could help to see what to expect when running the example, if those don't show up when running the example then there's something missing.

Reply:
The image makes sense there, but the full JSON trace I don't think has that much value. I think it's ok to expect people to run the sample if they want to see the big JSON output.